src/TLuaInterpreter.cpp

```diff
@@ -564,19 +563,18 @@ int TLuaInterpreter::isAnsiFgColor(lua_State* L)
 {
     int ansiFg;
 
-    std::string console = "main";
+    std::string windowName = "main";
 
     if (!lua_isnumber(L, 1)) {
-        lua_pushstring(L, "isAnsiFgColor: wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "isAnsiFgColor: bad argument #1 type (window name as string expected, got %s!)", luaL_typename(L, 1));
```

Comment:
Hang on - isn't the window name ***optional** in this one - omit it to get main profile console?

Reply:
See Discord